src/api.cc

```diff
@@ -698,10 +696,7 @@ class LocalLiteInstall : public LiteInstall {
 
   // TODO: implement `Install()` in such the way that `LiteClient` is not required as it is done for `Download()`
   InstallResult Install() override {
-    const auto tls_server{client_->config.tls.server};
-    client_->config.tls.server = "";
```

Comment:
I think with this change it doesn't make sense to override the `Install()` method at all, so the `LocalLiteInstall::Install()` can be removed at all.

Reply:
Removed.